packages/concerto-core/lib/decoratormanager.js

```diff
@@ -242,16 +297,88 @@ class DecoratorManager {
                 });
             }
         }
+    }
+
+    /**
+     * Adds decorator commands with index to the computed list
+     * @param {*} targetMap the target map to add the command to
+     * @param {targetKey} targetKey the target key to add the command to
+     * @param {DcsIndexWrapper[]} computedList the computed list to add the command to
```

Comment:
What's a computed list? The naming of this parameter and function aren't very clear to me.

Reply:
Fixed for all lists